### ISSUE TITLE
No refresh on local API

### ISSUE THREAD

**Issue by keamarg** [susanBuck/e28-spring21#32]
**Environment**
Local server API

**What operating system are you using?**
Mac

**Describe the problem**
/refresh is not working on my local API (works fine on production). It just keep adding new entries, reseeding the ones already in the seed files.

**Steps taken**
I have tried the bash setup command, /refresh, programmed an "axios" app to delete all the entries, but nothing seems to work. I am pretty sure it worked before, but maybe I should try deleting and rebuilding the API?

**Comment by susanBuck** [comment]
Can you share a screenshot of the following:

+ What you see when you visit `/refresh` locally
+ What you see when you run `bash setup` to rebuild the API locally

Can you also point me towards the code in your repo that's using axios to try and delete all the entries?

**Comment by keamarg** [comment]
Hi Susan

> What you see when you visit /refresh locally:

![image](https://user-images.githubusercontent.com/75420074/114917577-67104f00-9e26-11eb-8dea-7dec128aab70.png)

Also, an example of one of the ever-expanding tables:

![image](https://user-images.githubusercontent.com/75420074/114917691-85764a80-9e26-11eb-9783-8d987e5fccac.png)

> What you see when you run bash setup to rebuild the API locally:

![image](https://user-images.githubusercontent.com/75420074/114917771-9b840b00-9e26-11eb-9d37-c69bb8ae1494.png)

The "axios" code was just a "last resort" solution, where I tried to delete all of the entries manually, so it's not on the repo, and did not work (or it kinda worked until the next rebuild I think), along the lines of the following:

```javascript
 deleteAllProducts() {
      this.products.forEach(this.deleteProduct);
    },
    deleteProduct(item, index) {
      axios.delete("/product/" + index).then((response) => {
        if (response.data.errors) {
          console.log("nope");
          this.errors = Object.values(response.data.errors)[0][0];
          this.showConfirmation = false;
        } else {
          this.$emit("update-products");
          this.showConfirmation = true;
        }
      });
    },
    },
```

Thanks for helping out once again;)

**Comment by susanBuck** [comment]
Thanks for that extra info.

This is really odd. All signs indicate it should be successfully clearing the data. 

Let's try a "hard restart". To do this, remove the underlying core directory:

```bash
$ rm -rf /Applications/XAMPP/xamppfiles/htdocs/e28/e28api/core
```

Then cause it to be rebuilt by running the setup script:

```bash
$ bash setup
```

That should get you back to a clean slate. Once it does, try adding data and doing the `/refresh` or `bash setup` again and see if the same problem still exists.

**Comment by keamarg** [comment]
That worked like a charm! Should have done from the get go, but I was so sure it was me doing something wrong.

Thank you very much:D

**Comment by katz2row** [comment]
I know this was closed, and I am commenting after the fact, but thank goodness for this question. I had read it before when it was posted, but then forgot about it and just about lost my mind trying to figure out how to delete my bad table. I decided before posting to check if there were any threads on it (probably remembered this in the back of my mind), and there it was. Whoo hoo!!! Problem solved!
